src/reporters/csv.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2017 aixigo AG
+ * Released under the MIT license.
+ * https://github.com/aixigo/arestocats/blob/master/LICENSE
+ */
+
+module.exports = report;
+
+const fs = require( 'fs' );
+const path = require( 'path' );
+const { outcomes } = require( '../outcomes.js' );
+
+const reportsPath = 'reports';
+const resultsPath = path.join( reportsPath, 'results' );
+const csvDir = path.join( resultsPath, 'csv' );
+
+function report( allResults ) {
+   if( !fs.existsSync( reportsPath ) ) {
+      fs.mkdirSync( reportsPath );
+   }
+   if( !fs.existsSync( csvDir ) ) {
+      fs.mkdirSync( csvDir );
+   }
+   allResults.forEach( result => {
+      const testsuites = buildReport( result );
+      testsuites.forEach( summarizeSuite );
+      fs.writeFileSync( path.join( csvDir, `${result.subject.name}.csv` ), asCsv( testsuites ) );
```

Comment:
We should probably sanitize the `result.subject.name` to make sure that it does not contain `/` or (worse) `..`.

Reply:
good catch!